src/components/PoolOverview.tsx

```diff
@@ -153,13 +153,7 @@ export default function PoolOverview({
               <div className="margin Apr" key={symbol}>
                 {symbol.includes("/") ? (
                   <span className="label underline">
-                    <Tooltip
-                      title={
-                        <React.Fragment>
-                          {symbol.replaceAll("/", "\n")}
-                        </React.Fragment>
-                      }
-                    >
+                    <Tooltip title={symbol.replaceAll("/", "\n")}>
```

Comment:
great. you can also just wrap `ToolTip` around the span and not have to use the `<React.Fragment`.